vendor/lwd/dune-project

```diff
@@ -24,19 +24,6 @@
           (cbor :with-test) ; for the examples
           (containers :with-test))) ; for the examples
 
-(package
- (name tyxml-lwd)
```

Comment:
Removing the unnecessary parts will interfere with git subtree and can be avoided with `(vendored_dirs vendor)`: https://dune.readthedocs.io/en/stable/dune-files.html#vendored-dirs